ludwig/utils/data_utils.py

```diff
@@ -418,3 +419,11 @@ def default(self, obj):
             return obj.tolist()
         else:
             return json.JSONEncoder.default(self, obj)
+
+def generate_kfold_splits(data_train_df_fp, k_fold):
```

Comment:
this should probably get the DataFrame directly as input

Reply:
made modification to directly use DataFram as input to the `generate_kfold_splits()`